Dockerfile

```diff
@@ -2,6 +2,7 @@
 FROM node:lts-alpine as base-stage
 WORKDIR /app
 COPY package*.json ./
+RUN apk --no-cache --virtual build-dependencies add python make g++
```

Comment:
can you please clarify why we need this?

Reply:
This is needed to run jest and other testing-related dependencies. I've added a comment.